spec/controllers/api_flashcards/api/v1/review_controller_spec.rb

```diff
@@ -0,0 +1,130 @@
+require "rails_helper"
+require "shared_examples/shared_examples_for_api_controllers"
+require "support/helpers/api_helper.rb"
+include ApiHelper
+
+module ApiFlashcards
+  RSpec.describe Api::V1::ReviewController, type: :controller do
+    routes { ApiFlashcards::Engine.routes }
+
+    describe "GET#index" do
+      context "without credentials" do
+        before { get :index }
+        it_behaves_like "not authorized"
+      end
+
+      context "with incorrect credentials" do
+        before do
+          get :index,
+              request.headers["Authorization"] = encode("some@test.com",
+                                                        "nopass")
+        end
+        it_behaves_like "not authorized"       
+      end
+
+      context "with correct credentials" do
+        include_context "with correct credentials" do
+          let(:endpoint) do
+            {
+              verb: "get",
+              method: "index"
+            }
+          end
+        end
+        it "responds with 200 status code" do
+          expect(response.status).to eq(200)
+        end
+        it "responds with \'card_review\' key in response" do
+          expect(json_response.has_key?("card_review")).to eq(true)
+        end
+        it "responds with only one card" do
+          expect(json_response["card_review"]).not_to be_kind_of Array
+        end
+
+        %w(id original_text).each do |field|
+          it "contains \'#{field}\' key inside \'card_review\'" do
+            expect(json_response["card_review"].has_key?(field)).to eq(true)
+          end
+        end
+
+        it "does NOT contain \'translated_text\' key inside \'card_review\'" do
+          expect(json_response["card_review"].has_key?("translated_text")).to eq(false)
```

Comment:
Line is too long. [87/80]<br>`Hash#has_key?` is deprecated in favor of `Hash#key?`.